.github/workflows/portage-stable-packages-list

```diff
@@ -127,8 +127,6 @@ app-crypt/mhash
 app-crypt/mit-krb5
 app-crypt/p11-kit
 app-crypt/pinentry
-app-crypt/rhash
```

Comment:
rhash is a dependency of CMake, so something is amiss here?

Reply:
Bah, well spotted, thanks. Fortunately this does not warrant another CI run.

Reply:
Fixed.